docs/02.API-REFERENCE.md

```diff
@@ -3406,6 +3413,41 @@ jerry_create_string_sz (const jerry_char_t *str_p,
 - [jerry_create_string_from_utf8](#jerry_create_string_from_utf8)
 
 
+## jerry_create_regexp
```

Comment:
The code contains now two variants (`jerry_create_regexp` and `jerry_create_regexp_sz`). Both should be documented.